### DISCUSSION TITLE
Make the codebase more readable

### DISCUSSION THREAD

**Discussion by StefosGeo** [AI4Finance-Foundation/FinRL#315] [Ideas]
During my engangment with the repository so far, I realized that the codebase is not so readable. There are some easy and quick 
steps to fix this by using some linters and deleting unused code. I am willing to help on this if the administrators are agreed.

**Comment by BruceYanghy** [comment]
Sure, thanks a lot.

On Sun, Oct 17, 2021 at 10:52 PM Stefos Giorgos ***@***.***>
wrote:

> During my engangment with the repository so far, I realized that the
> codebase is not so readable. There are some easy and quick
> steps to fix this by using some linters and deleting unused code. I am
> willing to help on this if the administrators are agreed.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AI4Finance-Foundation/FinRL/discussions/315>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHR6TUQX64IXZ6KI3GK456TUHLPJZANCNFSM5GE4ZFMA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
*Hongyang(Bruce) Yang*
*Columbia University*
*Email: ***@***.*** ***@***.***> |
***@***.*** ***@***.***>*
*LinkedIn: **www.linkedin.com/in/brucehy
<http://www.linkedin.com/in/brucehy>*
